test/journeys/lib/test-helpers/index.js

```diff
@@ -53,38 +55,51 @@ export function moveMouse(aBrowser, selector) {
  * Reload each active sauce session to a fresh session
  * and name session according to suite
  * @param {string} name
+ * @param {object} browser
  * @returns {void}
  */
-export function renameJob(name) {
+export function renameJob(name, browser) {
+  if (!process.env.SAUCE) {
```

Comment:
Crap, I lied, I did do some refactors in this one. Just made it exit early if not on sauce 😄